app/components/dropdown-widget/template.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if (eq mode 'dropdown')}}
+    <div class="p-md">
+        <select class="form-control" onchange={{action "applyType" value="target.value"}}>
+            {{#each dropList key="@identity" as |type|}}
+                <option value={{type}} selected={{if (eq selectedType type) true}}>{{type}}</option>
+            {{/each}}
+        </select>
+    </div>
+{{/if}}
+
+{{#if (eq mode 'search')}}
+<div class="p-md">
+    {{#if enteredItem}}
+        <div class="facet-tag">Current selection:
+            <div><b>{{enteredItem}}</b> <span class="pointer" {{action "removeFilter" item.facetDashParameter}}>{{fa-icon "close"}} </span> </div>
+        </div>
+    {{/if}}
+    {{input class="form-control" value=filterText placeholder="Type to filter" key-up="filterVisible" focus-in="showList"}}
+    {{#if showList}}
+        <div class="dropdown-search">
+            {{#each filteredList as |item|}}
```

Comment:
It looks like identity is default now so it's not really needed anywhere here. I'll remove it.